### PR TITLE
PM representative responsibilities added

### PR DESCRIPTION
**Pull request by idvoretskyi**
With regards to the PM meeting [conversation](https://docs.google.com/document/d/1ZElyRqNsGebvMpikEBhSuf2Q6WrEYHB2PgWIlGgwt78/edit?ts=58740b3d#heading=h.1txgejyk87dr), SIG representative functions for the PM group are described.

### CODE REVIEW

**Comment by bgrant0607** [comment]
Thanks. We probably will need to refine this (e.g., which PM meetings?), but we can iterate. LGTM

**Comment by liggitt** [comment]
> which PM meetings

Agree. Tracking status, etc, seems reasonable, but in my experience PM meetings are rarely the most efficient way to accomplish that

**Comment by deads2k** [comment]
Was this discussed in some other pull, issue, or venue that I'm not aware of?   The first I heard about this was an email to sig-leads a couple hours ago, followed by this pull which merged without any discussion in less than 10 minutes.

This doesn't seem like an open way to make such a far reaching decision and it has the appearance of ignoring key stake holders in the decision.  I'm not aware of other sigs which have made (or appeared to make) unilateral process changes without a forum for discussion and relatively broad agreement.

@kubernetes/kubernetes-maintainers

File: governance.md [hunk 1]
```diff
@@ -137,6 +137,13 @@ In order to standardize Special Interest Group efforts, create maximum transpare
 * Participate in release planning meetings and retrospectives, and burndown meetings, as needed
 * Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc. 
 * Use the above forums as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
+* Represent the SIG for the PM group:
```
Comment (derekwaynecarr) [inline governance.md]: where this is in the doc is confusing, i would correlate that this is with a SIG-LEAD or whatever the new role is.  its not clear to me who in this document performs the below roles (versus the email).

Comment (derekwaynecarr) [inline governance.md]: So reading this closer, the net takeaway is that someone from each SIG must attend a PM group meeting of some kind at some interval?  I had initially read this as meaning that someone from the pm group would participate in each sig and not vice versa?

**Comment by pmorie** [comment]
> The first I heard about this was an email to sig-leads a couple hours ago, followed by this pull which merged without any discussion in less than 10 minutes.

+1

I do not think that names should have gone into a spreadsheet based on who happened to be on a particular call.  I would like the chance to digest what this means and discuss with the other leads of my SIG before we select a particular representative.

In general, when there is a representative needed from a SIG, I would propose that we do the following:

1.  Have a PR describing the role to the community repo
2.  Message on the sig-leads mailing list
3.  Not appoint people based on who happens to be present on a particular call

File: governance.md (hunk 1 above)
Comment (deads2k) [inline governance.md]: > So reading this closer, the net takeaway is that someone from each SIG must attend a PM group meeting of some kind at some interval? I had initially read this as meaning that someone from the pm group would participate in each sig and not vice versa?

There's a thread here suggesting the same thing: https://groups.google.com/forum/#!searchin/kubernetes-dev/inverted%7Csort:relevance/kubernetes-dev/pj5q4HjN8E8/u9qe_7NpFwAJ

Comment (idvoretskyi) [inline governance.md]: @derekwaynecarr yes, @deads2k has pointed to the right discussing thread.

**Comment by idvoretskyi** [comment]
@pmorie https://groups.google.com/forum/#!searchin/kubernetes-dev/inverted%7Csort:relevance/kubernetes-dev/pj5q4HjN8E8/u9qe_7NpFwAJ

**Comment by timothysc** [comment]
This is pretty contentious and I think needs to be brought before the community.  In the apache community, policy changes are voted on by the PMCs before enacted and debate is typically done on a separate list.  The PMCs often have the projects best interests in mind and will outline the reasons for the vote.  IMHO this type of policy shift requires approval from "a board" before enacted, b/c I think the process is ... yet another meeting.    

/cc @sarahnovotny

**Pull request merged**